modules/javafx.graphics/src/main/java/javafx/application/Preloader.java

```diff
@@ -114,6 +114,12 @@
         lineSeparator = prop != null ? prop : "\n";
     }
 
+    /**
+     * Creates a default {@code Preloader}.
+     */
+    public Preloader() {
+    }
```

Comment:
Not sure that "default" means anything here. I don't see any configuration.

Reply:
I didn't look closely at whether "default" was suitable in the previous cases. I like the JDK's wording, but in cases where there are constructors that allow configuration, the empty constructor could use "default", though specifying what the default values are is more beneficial anyway.